specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetworkGateway.json

```diff
@@ -1960,6 +1948,168 @@
         }
       },
       "description": "Response for ListLocalNetworkGateways API service call."
+    },
+    "virtualNetworkConnectionGatewayReference": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of VirtualNetworkGateway or LocalNetworkGateway resource."
+        }
+      },
+      "required": [
+        "id"
+      ],
+      "description": "A reference to VirtualNetworkGateway or LocalNetworkGateway resource."
+    },
+    "VirtualNetworkGatewayConnectionListEntityPropertiesFormat": {
```

Comment:
It utilizes `virtualNetworkConnectionGatewayReference` instead of `VirtualNetworkGateway` to match server response in `virtualNetworkGateways/{virtualNetworkGatewayName}/connections`'s GET example